backend/user-service/src/controllers/handlers/get-handler.ts

```diff
@@ -29,7 +29,7 @@ export const getUserById = async (request: Request, response: Response) => {
         id: userId,
       },
       include: {
-        preferences: {
+        Preferences: {
```

Comment:
Oh weird, the `preferences` should not be capitalised here. Refer to `schema.prisma` in the user service, it should be all lowercase.

Reply:
Hmm I'm not sure, when I tried to run the code there was an error and this was the suggested fix so I just changed it 😅 Did you manage to run it with lowercase?

Reply:
Ahh I see I see, okay thank you! Will resolve it when merging to the branch :)